Update play-random-video-from-folder.md

RealCaseyJones opened the pull request:
[![PR 180](https://badgen.net/badge/Preview/PR%20180/blue)](https://github.com/Streamer-bot-Extensions/Streamer.bot-Extensions-Wiki/pull/180) ![RealCaseyJones](https://badgen.net/badge/Author/RealCaseyJones/green) [![](https://badges.crowdin.net/streamer-bot-extensions-wiki/localized.svg)](https://translate.botextensions.dev/project/streamer-bot-extensions-wiki) ![](https://github.com/Streamer-bot-Extensions/Streamer.bot-Extensions-Wiki/actions/workflows/changelog-ci.yml/badge.svg) ![1724](https://badgen.net/badge/Description%20length/1724/green) [<img width="16" alt="Powered by Pull Request Badge" src="https://user-images.githubusercontent.com/1393946/111216524-d2bb8e00-85d4-11eb-821b-ed4c00989c02.png">](https://pullrequestbadge.com/?utm_medium=github&utm_source=Streamer-bot-Extensions&utm_campaign=badge_info)<!-- PR-BADGE: PLEASE DO NOT REMOVE THIS COMMENT -->

- [X] Included all images in documentation. (if using images please try use proper names)
- [X] Used relative pathing. i.e (/assets/hydrate/channelpointredeem.png)
- [X] Included import file. (not Code but File)
- [X] All Images/Files or anything else related go in there own folder inside "Assets"
- [X] Heading Names are the same as on the documentation writeup page
- [X] Made sure that all variables were camelCase (Non-camelCase variables won't work so won't be approved).
- [X] Detailed explanation of how your extension works (We test all of these so if we can't figure out how to use it the extension won't be approved).

Make sure to put an x in each check list item and that the items checked off have been completed.
Any extensions submitted with the checklist done and found not to be adhered to will be closed without warning.

RealCaseyJones committed:
Update play-random-video-from-folder.md

GoWMan813 committed:
Merge branch 'main' into patch-2

TerrierDarts reviewed (changes requested):
Hello, thanks for the updates. 
However a few changes are needed.
1. Files/Pictures need to be hosted locally, there is a folder called assets which they can be places in. (there is a folder in there for this extension and you can just place the files in there) ideally the file is in .sb format but thats neither here nor there)
2. If using an import file you will need to use this
`- [<i class="mdi mdi-file-download"></i> **Import Code *Click to Download***](/assets/20-sec-instant-replay/files/rewind-import.sb)
{.btn-grid .my-5}`
3. Description belongs as part of the header and not on the page
4.  optional configuration should be a secondary heading
5. finally if this is ws5 only you need the badge on there` *OBS WS v5*{.obs-version-badge}`

RealCaseyJones committed:
Add files via upload

RealCaseyJones committed:
Delete playRandomVideoFromFolder.sb

RealCaseyJones committed:
Add files via upload

RealCaseyJones committed:
Update play-random-video-from-folder.md

1. Uploaded assets/files to asset folder associated with this extension and updated documentation references
2. Updated formatting for import file
3. Removed 'Description' section from body of document and updated the header to reflect OBS WS v5 dependency
4. Updated 'Optional Configuration' heading format to reflect secondary heading syntax
5. Added OBS WS v5 badge

RealCaseyJones reviewed (commented):
Addressed all requested changes (see commit notes)

TerrierDarts reviewed (approved):
Approved.. Thank You
